### ISSUE TITLE
这是一个完整的ros 功能包吗

### ISSUE THREAD

**Issue by zyg234** [Zhutianyi7230/catkin_clocs#1]
请问，这是一个完整的ros 功能包吗

**Comment by Zhutianyi7230** [comment]
yes. It recieves the detection results from 2D and 3D detectors, and does the inference by `fusion_model.onnx `, and outputs the 3D detection results.

**Comment by zyg234** [comment]
您的检测器（YOLOP Pointpillars）结果是怎样输入进去的呢？
方便提供完整检测项目的代码吗，谢谢！

**Comment by Zhutianyi7230** [comment]
I put the result of YOLOP detections at `src\clocs\top_predictions.npy`. 
You can generate similar results using the official YOLOP code and some simple post-processing.
YOLOP I use is [https://github.com/hustvl/YOLOP](url)
